pkgs/by-name/eb/ebtks/package.nix

```diff
@@ -22,8 +22,10 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DLIBMINC_DIR=${libminc}/lib/cmake" ];
 
+  env.CXXFLAGS = lib.optionalString stdenv.cc.isClang "-Wno-register";
```

Comment:
I'm not very happy with this patch, I'll probably replace it with `postPatch` soon.